app/views/account/admin/users/_user.html.slim

```diff
@@ -1,7 +1,20 @@
-tr
-  td= user.email
-  td= user.full_name
-  td
-    = bann_user(user)
-  td
-    = admin_change_role(user)
\ No newline at end of file
+.table-responsive
+  table.table.table-hover
+    thead
+      tr
+        th= smart_listing.sortable 'Email', :email
```

Comment:
I am using this gem - https://github.com/Sology/smart_listing

Why not ok? This gem gives good functionality for searching and sorting

It also is integrated with kaminari pagination